hmda/src/main/scala/hmda/api/http/filing/submissions/ParseErrorHttpApi.scala

```diff
@@ -70,8 +71,8 @@ private class ParseErrorHttpApi(log: Logger, sharding: ClusterSharding)(implicit
             onComplete(fErrors) {
               case Success(state) =>
                 val parsingErrorSummary = ParsingErrorSummary(
-                  state.transmittalSheetErrors.map(parserErrorSummaryConvertor),
-                  state.larErrors.map(parserErrorSummaryConvertor),
+                  state.transmittalSheetErrors.map{hmdaParsedError => parserErrorSummaryConvertor(hmdaParsedError,submissionId.period)},
+                  state.larErrors.map{hmdaParsedError => parserErrorSummaryConvertor(hmdaParsedError,submissionId.period)},
```

Comment:
@lchen-2101  came up with a solution to reference an alternative valid value conf file.  let me know if this is too wild of an approach